test/render/test-cases.js

```diff
@@ -994,46 +958,6 @@ export const TEST_CASES = [
     ],
     referenceImageUrl: './test/render/golden-images/path-meter.png'
   },
-  {
-    name: 'mesh-lnglat',
-    viewState: {
-      latitude: 37.751537058389985,
-      longitude: -122.42694203247012,
-      zoom: 12,
-      pitch: 0,
-      bearing: 0
-    },
-    layers: [
-      new MeshLayer({
-        id: 'mesh-lnglat',
-        data: arrowDataLngLat,
-        mesh: new Arrow2DGeometry(),
-        sizeScale: 10000
-      })
-    ],
-    referenceImageUrl: './test/render/golden-images/mesh-lnglat.png'
-  },
-  {
-    name: 'mesh-lnglat-64',
-    viewState: {
-      latitude: 37.751537058389985,
-      longitude: -122.42694203247012,
-      zoom: 12,
-      pitch: 0,
-      bearing: 0
-    },
-    layers: [
-      new MeshLayer({
-        id: 'mesh-lnglat-64',
-        data: arrowDataLngLat,
-        coordinateSystem: COORDINATE_SYSTEM.LNGLAT_DEPRECATED,
-        fp64: true,
-        mesh: new Arrow2DGeometry(),
-        sizeScale: 10000
-      })
-    ],
-    referenceImageUrl: './test/render/golden-images/mesh-lnglat.png'
```

Comment:
why these tests are deleted?

Reply:
based on discussion with xiaoji, better to ignore mesh layer changes in this PR to avoid big PR and difficult review.

Reply:
Fine but add to follow-up tracker task